client/components/tinymce/plugins/wpcom-help/help-modal.jsx

```diff
@@ -135,6 +208,16 @@ class HelpModal extends React.Component {
 					{ this.getTableHead() }
 					<tbody>{ this.additionalShortcuts().map( this.renderRow, this ) }</tbody>
 				</table>
+				<p>{ spaceFormatText }</p>
+				<table className="wpcom-help__table">
+					{ this.getTableHead() }
```

Comment:
5 bonus awesome-points awarded for avoiding calling this repeatedly in here. ;)
Looking back through `getTableHead` and the functions it calls they're all ultimately 'static-ish' getters but by assigning it's value to a variable we do avoid extra calls to `translate`.